core/modules/flowgen.h

```diff
@@ -114,7 +114,8 @@ class FlowGen final : public Module {
   void PopulateInitialFlows();
 
   CommandResponse UpdateBaseAddresses();
-  bess::Packet *FillPacket(struct flow *f);
+  bess::Packet *FillUDPPacket(struct flow *f);
```

Comment:
Given that `FillUDPPacket` (btw, the style guide suggests `FillUdpPacket`, which I personally feel uglier) and `FillTcpPacket` share the considerable amount of logic, Could you merge them into `FillPacket` again but add a conditional for L4-specific routine inside the function?